editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -111,6 +111,14 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 			}
 		}
 
+		try {
+			Display.getDefault().syncExec( () -> postProjectCreationSetup(prj, subMonitor));
```

Comment:
you need to call subMonitor.newChild(1) and increase the number from 7 to 8 when creating the submonitor